api/proto/metrics/metrics.proto

```diff
@@ -192,10 +239,15 @@ message appMetric {
   string appVersion = 10;
   string AppName = 2;
   appCpuMetric cpu = 3;
+
+  // memory - OBSOLETE. Use memory_v2 instead
   memoryMetric memory = 4;
+
   repeated networkMetric network = 5;
   repeated appDiskMetric disk = 6;
   repeated appContainerMetric container = 7;
+
+  AppMemoryMetric memory_v2 = 8;
```

Comment:
Can we call this field appMemory instead?

Reply:
Done.